packages/strapi-plugin-upload/admin/src/components/CardControl/Wrapper.js

```diff
@@ -33,12 +33,24 @@ const Wrapper = styled.div`
         `;
       }
 
-      return `
+      if (type === 'plus') {
+        return `
           background-color: #AED4FB;
           border: 1px solid  #AED4FB;
           > svg {
             > g, path {
+              stroke: #007EFF;
```

Comment:
This color exists in the current theme so you can use the `mediumBlue` also for the `#AED4FB;`  can you replace it with `darkBlue`?

Reply:
Thanks !